src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -46,6 +46,7 @@ public class EditCommand extends Command {
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
+            + "[" + PREFIX_RELATIONSHIP + "RELATIONSHIP] "
```

Comment:
Hi, i think that relationship is made immutable so it shouldnt be able to be changed